types/typed-arrays.js

```diff
@@ -44,6 +44,7 @@ function create (TypedArray) {
             stateObj.buffers.push(buffer);
             return {
                 maxByteLength: buffer.maxByteLength,
+                resizable: buffer.resizable,
```

Comment:
only resizable one has maxByteLength property, so you can only store one of them to save some space

Reply:
Yeah, I thought about that, but figure it may be clearer even if not safer.